docs/getting-started/installing-substrate.md

```diff
@@ -2,37 +2,44 @@
 title: "Installing Substrate"
 ---
 
+Following ways to install Substrate assume you are running Mac OS, Arch, or a Debian-based OS like Ubuntu. For Windows users, you can try it [here](https://github.com/paritytech/substrate#612-windows).
+
+## Fast Installation
+
+If you plan to create your own blockchain, then run this simple one-liner:
```

Comment:
This is not so accurate.

The `--fast` flag can be used to simply install all the dependencies needed to build Substrate projects.

This includes:
- A package manager
- Open SSL
- CMAKE
- LLVM
- Substrate-Up scripts

You can see that all here: https://getsubstrate.io/

The only thing this wont do is compile and install a substrate node into your `cargo/bin`

Reply:
Great pickup.